ArduPlane/quadplane.cpp

```diff
@@ -1005,7 +1005,7 @@ void QuadPlane::check_yaw_reset(void)
 
 void QuadPlane::set_climb_rate_cms(float target_climb_rate_cms, bool force_descend)
 {
-    pos_control->input_vel_accel_z(target_climb_rate_cms, 0, force_descend);
+    pos_control->land_at_climb_rate_cm(target_climb_rate_cms, force_descend);
```

Comment:
This is not only used in landing. I think we should not change this line, instead update the callers that are landing. 

These two are the only ones that are landing I think:
https://github.com/ArduPilot/ardupilot/blob/6dcb137b29371dd49aadc546d4a672baad186210/ArduPlane/mode_qloiter.cpp#L106

https://github.com/ArduPilot/ardupilot/blob/6dcb137b29371dd49aadc546d4a672baad186210/ArduPlane/quadplane.cpp#L2807

Reply:
Yes, I agree. But it was strange the way plane references this through another call so I didn't want to do this without discussion. Any chance you could help with a commit here?

Reply:
Yeah, no problem.

Reply:
BTW, My understanding is these "pass-through" callers save flash. From a developer/code readability point of view I'm not a fan.

Reply:
@lthall Commit here: https://github.com/IamPete1/ardupilot/commit/711e35aa62b695500e017e00ec0b75f89d937e2d

Reply:
Thanks Pete!!

Reply:
Thanks Pete!